compiler/rustc_passes/src/stability.rs

```diff
@@ -164,8 +167,32 @@ impl<'a, 'tcx> Annotator<'a, 'tcx> {
 
         let (stab, const_stab) = attr::find_stability(&self.tcx.sess, attrs, item_sp);
 
-        let const_stab = const_stab.map(|(const_stab, _)| {
+        let const_stab = const_stab.map(|(const_stab, const_span)| {
             let const_stab = self.tcx.intern_const_stability(const_stab);
+            if let Some(fn_sig) = fn_sig {
+                if !(fn_sig.header.constness == hir::Constness::Const
+                    && fn_sig.header.abi != Abi::RustIntrinsic
+                    && fn_sig.header.abi != Abi::PlatformIntrinsic)
```

Comment:
Can all other ABIs be `const`? I'm not sure I fully understand but from reading https://github.com/rust-lang/rust/issues/69630#issuecomment-593436986 I thought we can only have `rustc_const_*` on `Abi::Rust` functions.

Reply:
This comment confused me a little too, as it does not make too much sense to have other ABIs for const fns as they are interpreted by the compiler... Anyone I should contact to get more information on this?

Reply:
It seems that `const extern` function will be supported in the future : #64926
So it's probably correct the way it is written right now!

Reply:
we can expand the list if we change the rules. being overeager in checking is never a problem for unstable attributes